docs/sources/maintaining/release-loki-build-image.md

```diff
@@ -9,22 +9,19 @@ is the Docker image used to run tests and build Grafana Loki binaries in CI.
 The build and publish process of the image is triggered upon a merge to `main`
 if there were made any changes in the folder `./loki-build-image/`.
 
-**Building and using the `loki-build-image` is a two-step process.**
+**Building and using the `loki-build-image` is a two-step process:**
 
-As a **first step** to build the new image, you need to create a pull
-request with the desired changes to the Dockerfile. To increase the version of
-the image, you also need to update the version tag of the `loki-build-image`
-pipeline defined in `.drone/drone.jsonnet` (search for
-`pipeline('loki-build-image')`) and run `DRONE_SERVER=https://drone.grafana.net/ DRONE_TOKEN=<token> make drone`
-and commit the changes to the same pull request.
-Once approved and merged to `main`, the image with the new version is built.
+## Step 1
 
-The new image can only be used after updating the `BUILD_IMAGE_VERSION` in the
-`Makefile` a **second step**. After changing the version in the Makefile and
-updating it in all other places where the image is used:
+1. create a branch with the desired changes to the Dockerfile
+2. update the version tag of the `loki-build-image` pipeline defined in `.drone/drone.jsonnet` (search for `pipeline('loki-build-image')`) to a new version number (try follow semver)
+3. run `DRONE_SERVER=https://drone.grafana.net/ DRONE_TOKEN=<token> make drone` and commit the changes to the same branch
+4. create a PR
+5. once approved and merged to `main`, the image with the new version is built
 
-* Dockerfiles in `cmd` directory
-* `.circleci/config.yml`
+## Step 2
 
-run `BUILD_IN_CONTAINER=false make drone` again and submit a PR with the
-generated changes.
+1. create a branch
+2. update the `BUILD_IMAGE_VERSION` variable in the `Makefile`
+3. run `loki-build-image/version-updater.sh <new-version>` to update all the references
+4. create a PR
```

Comment:
If we could make all steps dependent on `pipeline('loki-build-image')` I don't think we would need two steps :thinking: ... Ah, I think the issue was that we could not test it in a PR. Nevermind.